frontend/packages/multicluster-sdk/src/api/use-fleet-k8s-watch-resource/use-fleet-k8s-watch-resource.ts

```diff
@@ -143,11 +187,28 @@ export const useFleetK8sWatchResource = <R extends FleetK8sResourceCommon | Flee
     backendAPIPath,
   ])
 
-  const [defaultData, defaultLoaded, defaultError] = useK8sWatchResource<R>(useFleet ? null : resource)
+  let fallbackResource = null
+  if (!useFleet) {
+    fallbackResource = resource
+  }
+  const [defaultData, defaultLoaded, defaultError] = useK8sWatchResource<R>(fallbackResource)
```

Comment:
I like the original code better here... was there a Sonar complaint or something? There doesn't seem to be a good reason to use `let` here.